graphs/directed_and_undirected_(weighted)_graph.py

```diff
@@ -225,10 +225,9 @@ def has_cycle(self):
                                 anticipating_nodes.add(node[1])
                                 break
                             else:
+                                # FIXME: there used to be unreachable code here.
+                                # Code removed because of linter complaints.
```

Comment:
```suggestion
```
No point in having those comments. If this is creating a bug, people can open an issue.